guides/source/action_cable_overview.md

```diff
@@ -242,8 +242,8 @@ WebNotificationsChannel.broadcast_to(
 ```
 
 The `WebNotificationsChannel.broadcast_to` call places a message in the current
-subscription adapter (by default `redis` for production and `async` for development and
-test environments)'s pubsub queue under a separate broadcasting name for each user.
+subscription adapter's (by default `redis` for production and `async` for development and
+test environments) pubsub queue under a separate broadcasting name for each user.
```

Comment:
While we're editing this sentence structure I think it would read better as:

> The `WebNotificationsChannel.broadcast_to` call places a message in the current
subscription adapter's pubsub queue under a separate broadcasting name for each user.
The default pubsub queue for ActionCable is `redis` in production and `async` in development and
test environments.

What do you think?

Reply:
Good 👍 shall I update this PR?

Reply:
Done 👌